en/flight_modes/follow_me.md

```diff
@@ -1,23 +1,45 @@
 ---
-author: Jimmy Johnson
+author: Jimmy Johnson and Junwoo Hwang
 ---
 
 # Follow-Me Mode
 
 [<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
 
-*Follow Me* mode allows a multicopter to autonomously follow and track another system that is broadcasting its position using the [FOLLOW_TARGET](https://mavlink.io/en/messages/common.html#FOLLOW_TARGET) MAVLink message.
+*Follow Me* mode allows a multicopter to autonomously hold position and altitude relative to another system that is broadcasting its position (and optionally velocity) using the [FOLLOW_TARGET](https://mavlink.io/en/messages/common.html#FOLLOW_TARGET) MAVLink message.
 
-The vehicle will automatically yaw to face and follow the target from a specified [relative position](#NAV_FT_FS), [horizontal separation](#NAV_FT_DST) and [height](#NAV_MIN_FT_HT) above the home position.
-By default it will follow from directly behind the target at a distance of 8 meters, and a height of 8 meters above the home/arming position.
-While in this mode no user input is required.
+![Follow-me Concept](../../assets/flight_modes/followme_concept.png)
 
-:::tip
-PX4 currently ignores the altitude of/from the target and follows at a constant altitude above home.
-This limitation is because altitude sources from GPS on ground stations are often inaccurate.
+The vehicle will automatically yaw to face and follow the target from a specified [relative angle](#FLW_TGT_FA), [distance](#FLW_TGT_DST) and [height](#FLW_TGT_HT) and altitude, depending on the [altitude control mode](#FLW_TGT_ALT_M).
+
+By default it will follow from directly behind the target at a distance of 8 meters, and a height of 8 meters above the home (arming) position.
+
+Users can configure the follow angle, height and distance using a manual controller:
+- _Follow Height_ is controlled with the `up-down` input ("Throttle").
+  Center the stick to keep follow the target at a constant hight. Raise or lower the stick to adjust height.
+- _Follow Distance_ is controlled with the `forward-back` input ("Pitch").
+  Pushing the stick forward increases the follow distance, pulling it back decreases the distance.
+- _Follow Angle_ is controlled with the `left-right` input ("Roll").
+  The movement is from the user's perspective, so if you face the drone and move the stick left, it will move to your left.
+  From above if you move the stick left the drone will move counter-clockwise.
+
+  Follow Angle is defined as increasing in clockwise direction relative to the target's heading (which is 0 degrees)
+
+  ![Follow-me Angle Diagram](../../assets/flight_modes/followme_angle.png)
+
+
+:::note
+If you want to reset the RC-adjusted behavior, you need to either switch off from Follow-Me mode and activate again, or change the according [parameter](#parameters) to a different value.

```

Comment:
What does this mean? I.e. what behaviour would I reset and how? I looked at those parameters and they don't have anything for changing what RC input does what?